pkg/components/util/install.go

```diff
@@ -205,7 +205,16 @@ func UninstallComponent(c components.Component, kubeconfig []byte, deleteNSBool
 		}
 	}
 
+	releasesList, err := action.NewList(cfg).Run()
```

Comment:
Can we have e2e test installing 2 components into single namespace and then trying to remove them?

We could for example allow installing `httpbin` in separate namespace to make it easier, as this component should only create minimal amount of manifests (no not-namespaces resources etc).

Reply:
We can test for, headlamp as it is already installed in `lokomotive-system` namespace.

Reply:
Hm, but if we uninstall headlamp, then headlamp tests may fail. I guess we can run them as disruptive then.